### ISSUE TITLE
Move copy-webpack-plugin to dev dependencies

### ISSUE THREAD

**Issue by tobiasweibel** [harpreetkhalsagtbit/country-state-city#77]
I think the 3rd party copy-webpack-plugin needs to be moved to the dev dependencies as it's normally used for the build system.

https://github.com/harpreetkhalsagtbit/country-state-city/blob/140d6a166fdc039175c4ff7b3912342e6cc1c2b4/package.json#L62-L64

**Comment by lucasff** [comment]
agreed, why the hell I get a warning saying I need it as a peerDependencies ?
